packages/client/signers/src/communications/schemas.ts

```diff
@@ -1,20 +1,33 @@
 import { z } from "zod";
 
-const SupportedChainLayer = z.enum(["solana", "evm"]);
-const KeyType = z.enum(["secp256k1", "ed25519"]);
+const KeyTypeSchema = z.enum(["secp256k1", "ed25519"]).describe("Type of cryptographic key");
+const EncodingSchema = z.enum(["base58", "base64"]).describe("Encoding format for the key or data");
+
+const UserPublicKeySchema = z.object({
+    bytes: z.string().describe("The encoded public key value"),
+    encoding: EncodingSchema.describe("The encoding format of the public key payload"),
+    keyType: KeyTypeSchema.describe("The cryptographic algorithm of the public key"),
```

Comment:
Whatever we name this, we should be consistent internally (TEE, Crossmint Proxy) we rename the different pieces